ReadMe.md

```diff
@@ -119,12 +127,12 @@ called. It is a made up example of a Day-Night-Cycle system which switches betwe
 dispatches a game event via an `EventManager`.
 
 ```Kotlin
-class DayNightSystem(
-    private val eventMgr: EventManager
-) : IntervalSystem() {
+class DayNightSystem() : IntervalSystem() {
     private var currentTime = 0f
     private var isDay = false
 
+    private val eventMgr: EventManager = Inject.dependency()
```

Comment:
Out of curiosity as I saw something similar in other libraries: would it be possible to replace `Inject.dependency()` or `Inject.componentMapper()` by a `by inject()` statement? Similar to Kotlin's `by lazy()`?

Reply:
I agree with you. If this adds the overhead of an extra object/property and evaluates it every time then let's not do that for performance reasons ;)

Thanks for checking!